### PR TITLE
Initial upgrade docs: Skeleton creation

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

**Comment by swiftgist** [comment]
Looking at line 140 from the "Verify Steps from the Previous Upgrade" explains that if the customer has been using a custom ceph.conf that they really should convert it to the modular file sections in DeepSea.

Since this document is for SES7 and intended for the audience running SES5, the upgrade to SES6 is an intermediate step.  Using a custom ceph.conf will still work.  I think that the SES7 import step of the ceph.conf has to be sufficient so there's no need to send the customer through the exercise of separating the existing ceph.conf into Salt templates to take the resulting ceph.conf and giving it to the SES7 import step.  The original should work just as well; otherwise, the import process needs addressing.

As for the section, I suppose this section could actually say that the SES6 "convert your custom ceph.conf step" is unnecessary.

**Comment by asettle** [comment]
> Looking at line 140 from the "Verify Steps from the Previous Upgrade" explains that if the customer has been using a custom ceph.conf that they really should convert it to the modular file sections in DeepSea.
> 
> Since this document is for SES7 and intended for the audience running SES5, the upgrade to SES6 is an intermediate step. Using a custom ceph.conf will still work. I think that the SES7 import step of the ceph.conf has to be sufficient so there's no need to send the customer through the exercise of separating the existing ceph.conf into Salt templates to take the resulting ceph.conf and giving it to the SES7 import step. The original should work just as well; otherwise, the import process needs addressing.
> 
> As for the section, I suppose this section could actually say that the SES6 "convert your custom ceph.conf step" is unnecessary.

For future reference, the line in question is here: https://github.com/SUSE/doc-ses/pull/582/files#diff-412ce2e0102e6f0e8ffff5124bf89becR142

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -6,1852 +6,633 @@
...
       </para>
      </listitem>
      <listitem>
       <para>
-       Any used software packages have changed significantly.
+       Your password must be changed to meet &productname; &productnumber;
+       requirements. Ensure you change the username and password on
+       <emphasis>all</emphasis> initiators as well.
       </para>
      </listitem>
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:95]: not sure this is still relevant

File: xml/admin_ceph_upgrade.xml [hunk 2]
```diff
@@ -6,1852 +6,633 @@
...
      <listitem>
       <para>
-       Special precautions are necessary for your installation.
+       In case old RBD kernel clients (older than &prevcephos;) are being used,
+       refer to <xref linkend="rbd-old-clients-map" />. We recommend upgrading
+       old RBD kernel clients if possible.
       </para>
      </listitem>
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:102]: this might not be relevant at all

File: xml/admin_ceph_upgrade.xml [hunk 3]
```diff
@@ -6,1852 +6,633 @@
...
-<screen>/srv/salt/ceph/configuration/files/ceph.conf.import</screen>
+     </listitem>
+     <listitem>
       <para>
-       It is created by the engulf process during the upgrade from SES 4 to 5.
-       Also, the <option>configuration_init: default-import</option> option is
-       set in the file
+       You have to upgrade the &smaster; first, and replace &deepsea; with
+       <literal>ceph-salt</literal> and &cephadm; after.  You will be unable to
+       start using the &cephadm; orchestrator module until at least all &mgr;'s
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:113]:
```suggestion
       &cephsalt; and &cephadm; after.  You will <emphasis>not</emphasis> be able to
       start using the &cephadm; orchestrator module until at least all &mgr;'s
```

File: xml/admin_ceph_upgrade.xml [hunk 4]
```diff
@@ -6,1852 +6,633 @@
...
+        the upgrade.
       </para>
-<screen>/srv/pillar/ceph/proposals/config/stack/default/ceph/cluster.yml</screen>
-      <warning>
-       <title>Default &deepsea; Configuration</title>
+    </listitem>
+    </itemizedlist>
+  </sect2>
+
+   <sect2 xml:id="upgrade-backup-config-data">
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:142]: maybe referencing https://documentation.suse.com/ses/6/single-html/ses-admin/#cha-deployment-backup if it covers all needed info; if it does not, we should enhance the old version and republish it

File: xml/admin_ceph_upgrade.xml [hunk 5]
```diff
@@ -6,1852 +6,633 @@
...
-<screen>&prompt.sminion;systemctl start chronyd.service &amp;&amp; systemctl enable chronyd.service</screen>
-   </step>
-   <step>
-    <para>
-     Verify the status of chrony:
-    </para>
-<screen>&prompt.sminion;chronyc tracking</screen>
-   </step>
-  </procedure>
+ <sect1 xml:id="filestore2bluestore">
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:552]: SES6 doc have section on that as well, https://documentation.suse.com/ses/6/single-html/ses-deployment/#filestore2bluestore, reference sounds better

**Review by tbazant — CHANGES_REQUESTED** [review]
a few doubts, maybe not all are relevant

**Review by tbazant — APPROVED** [review]
goooood

**Pull request merged**